src/plugins/flexibleLayout/toolbarProvider.js

```diff
@@ -0,0 +1,189 @@
+
+class ToolbarProvider {
+    constructor(openmct) {
+        this.openmct = openmct;
+    }
+
+    provide() {
```

Comment:
My comment here should have said "...a function that returns an object, **or** preferably a class that you construct..." 

You went with "a function that returns an object **and** a class you construct.  Which doesn't quite make sense. 
 If you go with the class, then there's no point in the class having a `provide` method that creates a toolbarProvider.  The class should be the toolbar provider.  e.g. it should have the methods and properties you're returning in the provide method.

Reply:
I misunderstood your comment, I changed it to a function that returns an object

Reply:
👍